tutorials/README.md

```diff
@@ -3,3 +3,4 @@
 This repository includes a collection of official tutorials created and maintained by the Flame team. Use the index below to navigate through the available tutorials.
```

Comment:
I was thinking if we should slightly endorse the reader to follow them in sequence (not sure)

Reply:
Index per series?

Reply:
I like it to be index per series, but I think we can separate later, when we have more series, now we only have the basic ones.